man/systemd-sysext.xml

```diff
@@ -72,7 +72,10 @@
     <para>System extension images are strictly read-only by default. On mutable host file systems,
     <filename>/usr/</filename> and <filename>/opt/</filename> hierarchies become read-only while extensions
     are merged, unless mutability is enabled. Mutability may be enabled via the <option>--mutable=</option>
-    option; see "Mutability" below for more information.</para>
+    option or through the <varname>$SYSTEMD_SYSEXT_MUTABLE_MODE</varname> (for
```

Comment:
env vars are documented in docs/ENVIRONMENT.md, not in manpages

Reply:
Alright, moved docs to docs/ENVIRONMENT.md.